Sources/SwiftDocC/Benchmark/BenchmarkResults.swift

```diff
@@ -39,24 +39,23 @@ public struct BenchmarkResults: Codable {
         self.doccArguments = doccArguments
         self.metrics = Self.sortedMetrics(unorderedMetrics)
     }
-    
+
     /// A private convenience method for sorting metrics in a stable order based on their
     fileprivate static func sortedMetrics(_ unorderedMetrics: [Metric]) -> [Metric] {
         // Sort by value type and then by name
         return unorderedMetrics.sorted { (lhs, rhs) in
-            if lhs.value.kindSortOrder == rhs.value.kindSortOrder {
```

Comment:
Hmm this is an actual rewrite from if -> guard? Interesting I didn't think the formatter would make actual code changes.

Reply:
Its working on the AST so it can make "simple" structural changes

Reply:
I enabled the early exit setting, so it can do this. We can disable it if we want.